core/tests/test_utils.py

```diff
@@ -701,15 +803,19 @@ class FunctionWrapper(object):
     def __init__(self, func):
         """Creates a new FunctionWrapper instance.
 
-        args:
-          - func: a callable, or data descriptor. If it's a descriptor, its
+        Args:
+            func: a callable, or data descriptor. If it's a descriptor, its
             __get__ should return a bound method. For example, func can be a
```

Comment:
Indent this and the next line by 4 more spaces, since they are continuation lines.